prisma/schema.prisma

```diff
@@ -117,4 +118,17 @@ model Address {
   website   Website?  @relation(fields: [websiteId], references: [id])
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
-}
\ No newline at end of file
+}
+
+model Contact {
```

Comment:
remove these attributes: [office, social_networks, link_social_networks]

and apply these:

`{
   email
   phone
   name
   links: Link[]
}`


then:
`
model Link {
  id,
  url,
  label,
}
`